DAO.sol

```diff
@@ -401,6 +403,26 @@ contract DAO is DAOInterface, Token, TokenCreation {
         return true;
     }
 
+    function checkNewProposalRestrictions(
```

Comment:
Here we should include `transferFromWithoutReward` and `transferwithoutReward` to prevent 14% theft of the old DAO.